test/Microsoft.SqlTools.ServiceLayer.UnitTests/ObjectExplorer/ServerVersionHelperTests.cs

```diff
@@ -151,6 +160,17 @@ private void VerifyCalculateServerType(string serverVersion, SqlServerType expec
             Assert.Equal(expected, actual);
         }
 
+        private void VerifyCalculateServerTypeForEngineEdition(int engineEdition, SqlServerType expected)
+        {
+            ServerInfo serverInfo = new ServerInfo
+            {
+                EngineEditionId = engineEdition
+            };
+
+            SqlServerType actual = ServerVersionHelper.CalculateServerType(serverInfo);
+            Assert.Equal(expected, actual);
```

Comment:
please add messages for asserts. These failures are annoyingly hard to debug without them.

Reply:
Per offline chat - unfortunately there isn't support for adding message for Assert.Equal.

Reply:
Per offline chat - added workaround since Assert.Equal doesn't support adding messages.